web/src/scripts/webflow/methods.mjs

```diff
@@ -56,38 +56,33 @@ const getAllCollections = async () => {
 const createItem = (item, collectionId, draft = true) => {
   return axios({
     method: "post",
-    url: `https://api.webflow.com/collections/${collectionId}/items`,
+    url: `https://api.webflow.com/v2/collections/${collectionId}/items`,
     headers: { ...headers, "content-type": "application/json" },
     data: {
-      fields: {
-        _archived: false,
-        _draft: draft,
-        ...item,
-      },
+      isArchived: false,
+      isDraft: draft,
+      ...item,
```

Comment:
I know this because we talked about it, but for the purposes of making sure that other reviewers understand the change it might be helpful to make a comment in the review to the docs so people know why you changed the keys.

Reply:
I already had a link to the V2 docs, but I have now added the changes.

Reply:
i was gonna ask lol

Reply:
More specifically, when I'm making a change like this I sometimes try to explain the exact code change by leaving my own comments in the PR. Not just in the description. The description is great for explaining high level approach. If you linked the docs in the description it might be helpful to write a comment in the PR like:

"These keys were updated in version 2 of the Webflow API. See documentation linked in description for a complete list of changes."

Reply:
I'm calling this out not because anything is wrong, but leaving comments like that in GH makes it easier for other reviewers and helps you as the author to re-consider all the changes you made.